### DISCUSSION TITLE
Recent changes

### DISCUSSION THREAD

**Discussion by steve1977** [RandomNinjaAtk/docker-lidarr-extended#39] [Q&A] [answered]
I am using lidarr-extended on unraid via docker.

I saw the download directory was changed / updated, but haven't seen this change in my docker settings. Is this expected or do I need to purge the docker and start from scratch to see it?

**Comment by RandomNinjaAtk** [comment, marked as answer]
I don't think unraid has the capability to update the template after initial setup. So you either have to manually update it, or deleting and reinstalling should work in theory, just make sure you use the same volume mappings.

**Comment by steve1977** [comment]
Makes sense. Let me look into this. Is there anything that has changed in the container settings over the last month besides the download folder?

**Reply by Bait-Fish** [reply]
Or compare what you have currently in Unraid versus https://github.com/RandomNinjaAtk/unraid-templates/blob/master/randomninjaatk/lidarr-extended.xml including the history listed there for changes. That's what I did.

**Comment by steve1977** [comment]
I wiped and reinstalled the container. It now has the new download directory, but no longer has the "old" download container folder mapped, which I thought was still used for the usenet downloads. Is this intentional? I thought there would now be two download folders being mapped?

**Reply by RandomNinjaAtk** [reply]
" no longer has the "old" download container folder mapped"

The old volume is no longer a required volume, so it's not in the template. See the readme...

**Comment by steve1977** [comment]
Checked the readme, but not fully clear. If I like to continue to also use usenet download, do I need to manually add it?

**Reply by RandomNinjaAtk** [reply]
Uhh.... its kinda up to you what you want to do....

**Comment by steve1977** [comment]
I don't want to solely rely on tidal/deezer, but still like to use usenet download (as done by the original lidarr). For this purpose, I'll need two separate download directories? And for this purpose manually map it?

**Reply by Protektor-Desura** [reply]
Lidarr has built in support for Usenet. So these download scripts don't change that Lidarr supports internally both bittorrent and usenet.

**Comment by steve1977** [comment]
Thanks. I know I know. Just wondering whether I can do so into the same download folder or I'd need to map a separate one for usenet downloads?

Or in other words, why was the mapping of the download folder even change in the first case if I can now use the newly mapped one for both tidal/deezer and usenet/torrent?

**Reply by Protektor-Desura** [reply]
Usenet downloads are normally is a separate download directory of it's own. Then the *arrs talk to NZBGet or whatever to know when they have finished downloading and are ready for processing, just like bittorrents.
